prometheus/datadog_checks/prometheus/data/conf.yaml.example

```diff
@@ -82,5 +82,9 @@ instances:
   # The path to the trusted CA used for generating custom certificates
   # ssl_ca_cert: "/path/to/cacert"
 
+  # The check limits itself to 2000 metric contexts (different metric name or different tags)
```

Comment:
Can we get away with "metrics" rather than "metric contexts"?  We call the unit of billing "custom metrics" even though we count contexts, and we have a KB on the calculation.  Otherwise, if "timeseries" is equivalent to "contexts" lets use that.

Reply:
Also the config is "max_returned_metrics" so that tracks better